game.py

```diff
@@ -547,10 +529,11 @@ def _new_dot_gameover(self, color):
         self._svg_width = self._dot_size_gameover
         self._svg_height = self._dot_size_gameover
         return svg_str_to_pixbuf(
-            self._header() + \
-            self._circle(self._dot_size_gameover / 2., self._dot_size_gameover / 2.,
-                         self._dot_size_gameover / 2.) + \
-            self._footer())
+                            self._header()
+                            + self._circle(self._dot_size_gameover / 2.,
+                                           self._dot_size_gameover / 2.,
+                                           self._dot_size_gameover / 2.)
+                            + self._footer())
```

Comment:
I like the change here but can you keep the old indentation, I don't see a reason for increasing it this way.

Reply:
if I just placed the operator before the next operand keeping the old indentation, line 533 would cross 79 character per line limit and the next line would be over-indented. Hence I had to change the indentation this way.

Reply:
I don't see how that'll be the case as keeping the old indentation would look something like this;
```python3
def _new_dot_gameover(self, color):
    ''' generate a dot of a color color '''
    self._stroke = color
    self._fill = color
    self._svg_width = self._dot_size_gameover
    self._svg_height = self._dot_size_gameover
    return svg_str_to_pixbuf(
        self._header()
        + self._circle(self._dot_size_gameover / 2.,
                       self._dot_size_gameover / 2.,
                       self._dot_size_gameover / 2.)
        + self._footer())
```

Reply:
Oh my bad, I had over indented line 534 and 535 when I tried keeping the old indentation n tested for flake errors, so I'd got over indentation error previously😅